CHANGELOG.md

```diff
@@ -14,6 +14,16 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+
+## [2.34.1] - 2021-11-22
+### Added
+- `transformations` api client, which allows the creation, deletion, update and retrieval of transformations.
+- `transformations.schedules` api client, which allows the schedule, unschedule and retrieval of recurring runs of a transformation.
+- `transformations.notifications` api client, which allows the creation, deletion and retrieval of transformation email notifications.
+- `transformations.schema` api client, which allows the retrieval of the expected schema of sql transformations based on the destination data type.
+- `transformations.jobs` api client, which retrieves the  status of transformation runs.
+- `transformations.run` method, which runs transformations.
```

Comment:
I think this can be included in the first line along with "deletion, update, run and retrieval"...